### PR TITLE
Added Audio for Introduction of structure in C

### LINKED ISSUE #6979: Audio for Introduction to structure in C
### Description

I will do audio contribution on the topic Introduction to structure in C.

### Domain

C/CPP

### Type of Contribution

Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Harsh652-cpu**
<hr>

## Description 📜

In C langauge we come across some cases when we need to use multiple datatypes and its variables simultaneously.Structure is used for such cases.In this audio a brief overview of an Introduction to structure is given.I hope it will add some values in your coding journey. 
Fixes #6979 

<hr>

## Type of change 📝

- [x] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Harsh652-cpu** [comment]
@Suvraneel Have a look at this pr.

File: C_CPP/Structure padding, Union, Enum, & typedef/Audio_Introduction_to_structure.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# INTRODUCTION TO STRUCTURE
+
+*In this audio file,I have covered following topics:-*
+<br>
```
Comment (apeksha235) [inline C_CPP/Structure padding, Union, Enum, & typedef/Audio_Introduction_to_structure.md:4]: remove this br tag

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@Harsh652-cpu Amazing!!! Just make this change and you're good to go

File: C_CPP/Structure padding, Union, Enum, & typedef/Audio_Introduction_to_structure.md (hunk 1 above)
Comment (Harsh652-cpu) [reply C_CPP/Structure padding, Union, Enum, & typedef/Audio_Introduction_to_structure.md]: Done @apeksha235 please check now.

**Comment by Harsh652-cpu** [comment]
Done @apeksha235 Please review it now.

**Comment by Saviour1001** [comment]
Looks great 🥳

**Pull request merged**